kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -83,6 +77,7 @@
   private boolean useMessageTimeDatePartitioning;
 
   private TopicPartitionManager topicPartitionManager;
+  private SinkRecordFilter sinkRecordFilter;
```

Comment:
did you mean to put the filter logic in this PR?

Reply:
Sorry about the confusion... This PR is branched off the filter&sanitize branch and has changes in the other PR.